server/util/src/api_helper.rs

```diff
@@ -0,0 +1,42 @@
+use std::time::Duration;
+
+use reqwest::*;
+
+pub struct RetrySeconds(Vec<u64>);
+
+impl Default for RetrySeconds {
+    fn default() -> Self {
+        Self(vec![
+            /* first retry */ 2, /* second retry */ 5, /* third retry */ 10,
+        ])
+    }
+}
+
+pub async fn with_retries<F>(connection_timeouts: RetrySeconds, f: F) -> Result<Response>
+where
+    F: Fn(Client) -> RequestBuilder,
+{
+    let mut index = 0;
+    loop {
+        let client = Client::builder()
+            .connect_timeout(Duration::from_secs(connection_timeouts.0[index]))
+            .build()
+            .unwrap();
```

Comment:
```suggestion
            .unwrap();// This method fails if a TLS backend cannot be initialized, or the resolver cannot load the system configuration.
```

Might nice to put a comment on the unwrap? I think it's safe enough though.

Reply:
Agree, thanks